.github/workflows/macos-ci.yml

```diff
@@ -45,8 +45,13 @@ jobs:
       # - libpng
       # - libvorbis
       # - python
+      - name: Setup Python
+        uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
+        with:
+          python-version: 3.12.8
```

Comment:
This specific Python version worked. 3.13 did not work.

Reply:
I'm fine with this for now, but not supporting newer versions is a bad long term strategy. We should document this.

Reply:
@royfalk Agreed.